embedded/android/gradlew.bat

```diff
@@ -70,11 +70,11 @@ goto fail
 :execute
 @rem Setup the command line
 
-set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
```

Comment:
This looks a bit surprising. Maybe we can have @hughns checking this before we merge.

Reply:
thx